toolkit/docs/formats/imageconfig.md

```diff
@@ -416,58 +416,9 @@ A sample KernelOptions specifying a default kernel:
-
-##### Separate `/boot` Partition
-
-Since the root partition's hash tree is stored as part of the initramfs, the initramfs cannot be stored on the same root partition (it would invalidate the measurements). To avoid this a separate `/boot` partition is needed to house the hash tree (via the initramfs).
-
-##### ISO
-
-The ISO command line installer supports enabling read-only roots if they are configured through the configuration JSON file (see [full.json's](../../imageconfigs/full.json) `"Azure Linux Core Read-Only"` entry). The automatic partition creation mode will create the required `/boot` partition if the read-only root is enabled.
-
-The GUI installer does not currently support read-only roots.
-
-- `Enable`: Enable dm-verity on the root filesystem
-- `Name`: Custom name for the mounted root (default is `"verity_root_fs"`)
-- `ErrorCorrectionEnable`: Enable automatic error correction of modified blocks (default is `true`)
-- `ErrorCorrectionEncodingRoots`: Increase overhead to increase resiliency of the forward error correction (default is `2` bytes of code per 255 bytes of data)
-- `RootHashSignatureEnable`: Validate the root hash against a key stored in the kernel's system keyring. The signature file should be called `<Name>.p7` and must be stored in the initramfs. This signature WILL NOT BE included automatically in the initramfs. It must be included via an out of band build step.
-- `ValidateOnBoot`: Run a validation of the full disk at boot time, normally blocks are validated only as needed. This can take several minutes if the disk is corrupted.
-- `VerityErrorBehavior`: Indicate additional special system behavior when encountering an unrecoverable verity corruption. One of `"ignore"`, `"restart"`, `"panic"`. Normal behavior is to return an IO error when reading corrupt blocks.
-- `TmpfsOverlays`: Mount these paths as writable overlays backed by a tmpfs in memory.
-- `TmpfsOverlaySize`: Maximum amount of memory the overlays may use. Maybe be one of three forms: `"1234"`, `"1234[k,m,g]"`, `"20%"` (default is `"20%"`)
-- `TmpfsOverlayDebugEnabled`: Make the tmpfs overlay mounts easily accessible for debugging purposes. They can be found in /mnt/verity_overlay_debug_tmpfs. Include the
-    `verity-read-only-root-debug-tools` package to create the required mount points.
-
-A sample ReadOnlyVerityRoot specifying a basic read-only root using default error correction. This configuration may be used for both normal images and ISO configurations:
-
-``` json
-"ReadOnlyVerityRoot": {
-    "Enable": true,
-    "TmpfsOverlays": [
-        "/var"
-    ],
-},
-```
+See [Azure Linux Image Customizer configuration](../../tools/imagecustomizer/docs/configuration.md#verity-type) for more information.
```

Comment:
Is this a GA blocker as well? Technically, we shouldn't deprecate features after we GA. Or was that already considered and the risk is of breaking anyone is negligible?

Reply:
https://github.com/microsoft/azurelinux/pull/9537 should have disabled support already, I just forgot to remove this bit of the docs.